package.json

```diff
@@ -7,7 +7,7 @@
     "scripts": {
         "foreach-with-fallback": "yarn workspaces foreach --verbose --topological exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js $0 $1",
         "postinstall": "yarn run build",
-        "start-dev": "yarn workspaces foreach --verbose --parallel --jobs unlimited --interlaced exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js start-dev global:start-dev",
+        "start-dev": "yarn run build && yarn workspaces foreach --verbose --parallel --jobs unlimited --interlaced exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js start-dev global:start-dev",
```

Comment:
**Downside:** One downside to this approach is that if `yarn run build` fails (e.g., due to a typescript error) then dev mode won't successfully start. 

If errors happen after the initial build during development, everything is fine since the dev servers don't mind that.